src/jsontestrunner/main.cpp

```diff
@@ -56,7 +56,7 @@ static Json::String readInputTestFile(const char* path) {
     return "";
   fseek(file, 0, SEEK_END);
   long const size = ftell(file);
-  size_t const usize = static_cast<unsigned long>(size);
+  const auto usize = static_cast<size_t>(size);
```

Comment:
All this buggy fuss.
Meanwhile this whole function is conceptually 4 easy lines. :)

```
static Json::String readInputTestFile(const char* path) {
    std::ifstream in(path, in.binary);
    std::stringstream os;
    os << in.rdbuf();
    return Json::String(os.str());
}
```

Reply:
Yes, that's great! but this sould be in another PR.